typings/storybook__addons/index.d.ts

```diff
@@ -35,6 +36,10 @@ declare module '@storybook/addons' {
          */
         excludeDecorators: boolean;
       };
+
+      container: React.ComponentType<any>;
+      page: React.ComponentType<any>;
```

Comment:
Config of Storybook has additional options 😯
Suggestions for improving types are welcomed

Reply:
can we use `unknown` rather than `any` ?

Reply:
`unknown` throws:

![image](https://user-images.githubusercontent.com/14183168/143592302-5cf79f82-0a44-4327-b2a6-a6124607de9d.png)

---

Also tried:

```tsx
(props: unknown) => React.ReactElement
```

![image](https://user-images.githubusercontent.com/14183168/143592658-cfcaf2d6-aa8d-46a6-b8e4-816b5919d25f.png)

Reply:
I was afraid you gonna experience that - its the nature how react types are authored, bottom types and `any` all over the place :(, ok lets keep it as is.

Additional thought: can you maybe remove explicit generic -> `any`, as that type has a default generic - the omnipresent former bottom type `{}` ?

Reply:
> Additional thought: can you maybe remove explicit generic -> any, as that type has a default generic - the omnipresent former bottom type {} ?

```
$ tsc -b tsconfig.json
.storybook/preview.js:37:5 - error TS2322: Type 'FC<FluentDocsContainerProps>' is not assignable to type 'FunctionComponent<{}> | ComponentClass<{}, any> | undefined'.
  Type 'FunctionComponent<FluentDocsContainerProps>' is not assignable to type 'FunctionComponent<{}>'.
    Types of parameters 'props' and 'props' are incompatible.
      Type '{ children?: ReactNode; }' is not assignable to type 'PropsWithChildren<FluentDocsContainerProps>'.
        Property 'context' is missing in type '{ children?: ReactNode; }' but required in type 'FluentDocsContainerProps'.
```

Reply:
well :D it isssss what it is. thanks!